Makefile

```diff
@@ -5,13 +5,17 @@ NBIN=./node_modules/.bin
 WEBPACK=$(BIN)/webpack.sh
 MOCHA=$(NBIN)/mocha
 ESLINT=$(NBIN)/eslint
+WEBPACK_DEV_SERVER=$(NBIN)/webpack-dev-server
+NPM_CHECK_UPDATES=$(NBIN)/ncu
 
 help:
 	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
 
 
 badger: ## Red Badger 4 lyfe!
+	@echo ''
 	@cat assets/badger.txt
+	@echo ''
```

Comment:
We can cross that bridge if we need to :) I just think it'd look tidier in the file itself.
Of course, this doesn't block the PR going through